content/governance.md

```diff
@@ -66,7 +66,7 @@ The current team members are:
 * Matt Layher ([Fastly](https://www.fastly.com/))
 * Matthias Rampke ([SoundCloud](https://soundcloud.com/))
 * Max Inden ([Parity Technologies](https://www.parity.io/))
-* Richard Hartmann ([SpaceNet](https://space.net/))
+* Richard Hartmann ([Grafana Labs](https://www.grafana.com/))
```

Comment:
```suggestion
* Richard Hartmann ([Grafana Labs](https://grafana.com/))
```

Your colleagues don't have the www. I suggest removing it so if someone ever greps the list they can easily sort by company.